src/components/order-service/order-service-review.html

```diff
@@ -53,18 +61,18 @@ <h3 class="results-message">
   <div ng-if="$ctrl.orderComplete && $ctrl.bindType === 'none'">
     <p><a ng-href="{{$ctrl.selectedProject | projectUrl : $ctrl.baseProjectUrl}}">Continue to the project overview</a> to bind this service to your application. Binding this service creates a secret containing the information necessary for your application to use the service.</p>
   </div>
-  <div ng-if="!$ctrl.error">
-    <p class="or">or</p>
+  <div ng-if="$ctrl.serviceClass.resource.externalMetadata.documentationUrl || $ctrl.serviceClass.resource.externalMetadata.supportUrl || $ctrl.serviceInstance.status.dashboardURL">
+    <p class="or" ng-if="!$ctrl.error">- or -</p>
     <p>Browse resources for {{$ctrl.serviceClass.name}}:</p>
```

Comment:
I think this will show up with nothing below if there an error and the service ONLY has a dashboard URL